README.md

```diff
@@ -153,6 +154,16 @@ And it requires annoy steps to generate, sets and maintains tokens [even if refi
 
 This action provides another way. It checks other workflows/jobs statuses in actions with GITHUB_TOKEN.
 
+## Support for Github Enterprise
+
+To run this action in your Github Enterprise (GHE) instance you need to override `github-api-url`:
+
+``` yaml
+with:
+  github-api-url: "https://ghe-host.acme.net/api/v3"
+
```

Comment:
```suggestion
  github-api-url: "https://ghe-host.example.net/api/v3"
```

I didn't know detail about acme wording 🙇 However the domain looks like showing domain sales. How about to use example domains?

Reply:
Thank you, applied!